lib/crypto/packaged.py

```diff
@@ -115,7 +118,7 @@ def _no_sign(src, dest_path):
     # If this is a local development instance, just copy the file around
     # so that everything seems to work locally.
     log.info('Not signing the app, no signing server is active.')
-    with storage.open(dest_path, 'w') as dest_f:
+    with private_storage.open(dest_path, 'w') as dest_f:
```

Comment:
Unless the comment above is wrong and it's sometimes used in a different context ? I hope not :)